README.md

```diff
@@ -3,23 +3,19 @@
 This repo contains the Ruby gem that distributes the [Tech Docs Template][tdt-template]. The Tech Docs Template is a [middleman template][mmt] that
 you can use to build technical documentation using a GOV.UK style.
 
-👉 Find out more about the template and its features from the [Tech Docs Template documentation][tdt-docs].
-
-## Usage
-
-👉 Find out how to [generate a new website with the Tech Docs Template][tdt-readme].
+👉 To find out more about setting up and managing content for a website using this template, see the [Tech Docs Template documentation][tdt-docs].
```

Comment:
Would be good to remove the emojis if poss. I think they cause accessibility issues.